README.md

```diff
@@ -29,29 +63,128 @@ A valid YAML steps sequence looks like this:
         - No thanks: null
 ```
 
+### Sequence of Bot Messages
 
-Installation
-------------
+Each step can contain more than one bot messages. Multiple bot messages are displayed
+in succession.
 
-Install the requirements into the Python virtual environment of your
-`edx-platform` installation by running the following command from the
-root folder:
+```yaml
+- step1:
+    messages:
+        - Let's do some math!
+        - We will start with simple sums first.
+        - What is the sum of 1 and 1?
+    responses:
+        - 2: step2
+        - 3: step3
+- ...
+```
 
-```bash
-$ pip install -r requirements.txt
+### Step without Bot Messages
```

Comment:
Note that steps without bot messages are implemented by #2 and will only start working correctly when #2 is merged. I included the documentation here instead of #2 to avoid merge conflicts due to the changes in the structure of the README doc I made in this PR.